src/commands/nfts/burnNft.ts

```diff
@@ -17,7 +16,9 @@ export const burnNft = async (
   config: ConfigEntry,
   logger: Logger
 ): Promise<ExecutionOutput> => {
-  const { verbose, did} = argv
+  const { verbose, did, nftType} = argv
+
+  const tokenId = nftType == 721 && argv.tokenId ? argv.tokenId : argv.did
```

Comment:
```suggestion
  const tokenId = nftType == 721 && (argv.tokenId || argv.did)
```
it can be reduced like this

Reply:
for me personally the current line is more readable, but I guess it's subjective